examples/counter-anchor/README.md

```
@@ -0,0 +1,64 @@
+# counter-anchor
+
+A minimal **Anchor** program demonstrating the full Ephemeral Rollups lifecycle:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)